packages/frontend/src/hooks/useFilters.ts

```diff
@@ -0,0 +1,100 @@
+import {
+    Dimension,
+    DimensionType,
+    Field,
+    FilterGroup,
+    FilterGroupOperator,
+} from 'common';
+import { defaultValuesForNewStringFilter } from '../filters/string-filter/StringFilterForm';
+import { useExplorer } from '../providers/ExplorerProvider';
+
+export const useFilters = () => {
+    const {
+        state: { filters: activeFilters },
+        actions: { setFilters },
+    } = useExplorer();
+
+    const addFilterGroup = (filterGroup: FilterGroup) =>
+        setFilters([filterGroup, ...activeFilters]);
```

Comment:
Personal opinion: I would expect the new filter to be at the end of the list instead of the beginning.

Reply:
fixed